Buildings/Controls/OBC/Utilities/PIDWithAutotuning/Relay/BaseClasses/NormalizedTimeDelay.mo

```diff
@@ -55,6 +55,10 @@ protected
   Buildings.Controls.OBC.CDL.Logical.Not and1
     "Disable the assert when the tuning is not ongoing"
     annotation (Placement(transformation(extent={{28,-90},{48,-70}})));
+  Buildings.Controls.OBC.CDL.Logical.TrueDelay tunEndDel(
```

Comment:
Is this instance needed? I am not comfortable of adding the random delay time. User may ask why we use 0.001 seconds, not 0.0001 seconds.